webapp/src/main/scala/org/allenai/common/webapp/InfoRoute.scala

```diff
@@ -37,14 +39,14 @@ class InfoRoute(val info: Map[String, String] = Map.empty) {
       pathEndOrSingleSlash {
         respondWithMediaType(MediaTypes.`application/json`) {
           complete {
-            info.toJson.prettyPrint
+            infoWithTime.toJson.prettyPrint
```

Comment:
You could also just add it to the `withVersion` implementation (maybe renaming to `withVersionAndStartupTime` if you're feeling verbose).